frontend/device/generic/device.lua

```diff
@@ -84,11 +84,11 @@ local Device = {
     isAlwaysPortrait = no,
     -- On some devices (eg newer pocketbook) we can force HW rotation on the fly (before each update)
     -- The value here is table of 4 elements mapping the sensible linux constants to whatever
-    -- nonsense the device actually has. Canonically it should be { 0, 1, 2, 3 } if the device
+    -- nonsense the device actually has. Canonically it should return { 0, 1, 2, 3 } if the device
     -- matches <linux/fb.h> FB_ROTATE_* constants.
     -- See https://github.com/koreader/koreader-base/blob/master/ffi/framebuffer.lua for full template
     -- of the table expected.
-    usingForcedRotation = nil,
+    usingForcedRotation = function() return nil end,
```

Comment:
```suggestion
    usingForcedRotation = function() return end,
```

Reply:
(intentional, to signal there's meant to be no table)